Makefile.am

```diff
@@ -155,7 +156,7 @@ cpp_server_ct_server_LDADD = \
 	cpp/libcore.a \
 	$(json_c_LIBS) \
 	$(libevent_LIBS) \
-	-lglog -lgflags -lcrypto -lprotobuf -lpthread -lsqlite3
+	-lglog -lgflags -lcrypto -lprotobuf -lpthread -lsqlite3 -lleveldb
```

Comment:
From what I can see on Travis, you'll need another library everywhere you need `-lleveldb`?